### PR TITLE
Reference paper-template in documentation

### LINKED ISSUE #61: Add information about project/paper template repository to documentation
The repo is https://github.com/PennLINC/paper-template. It's a work in progress, but contains useful elements and recommendations.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #61.

Changes proposed:

1. Move "Contributing" folder to "documentation".
2. Remove `Contributing/ProjectTemplate.md`. This is now in the `gh-pages` branch of `paper-template`.
3. Overhaul `LabHome/ProjectSetup.md` and `documentation/project-documentation.md` to reference the `paper-template` repo.

Unfortunately, I needed to include the changes from #77 in order to render the website locally, so I'd recommend reviewing and merging #77 before this in order to reduce the diff.

### CODE REVIEW

File: docs/LabHome/TravelGuidelines.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ nav_order: 8
...
 
-As of 2024 PennLINC graduate students, post-docs, and senior scientists in good standing are entitled to $4k of funding for allowable academic travel per year. We no longer require that this travel be linked to a poster presentation or talk; we are all adult learners and I trust team members to evaluate what travel is most valuable for their development.  To ensure transparency and equity in travle expenses, please log your expenses [here](https://docs.google.com/spreadsheets/d/1fAKrIlrCRXMc8PNQwpw-Adb3GG91zLkvc1RoutVXzK0/edit?usp=sharing) as they are accrued. 
+As of 2024 PennLINC graduate students, post-docs, and senior scientists in good standing are entitled to $4k of funding for allowable academic travel per year. We no longer require that this travel be linked to a poster presentation or talk; we are all adult learners and I trust team members to evaluate what travel is most valuable for their development.  To ensure transparency and equity in travle expenses, please log your expenses [here](https://docs.google.com/spreadsheets/d/1fAKrIlrCRXMc8PNQwpw-Adb3GG91zLkvc1RoutVXzK0/edit?usp=sharing) as they are accrued.
```
Comment (kahinimehta) [inline docs/LabHome/TravelGuidelines.md:12]: travle --> travel

File: docs/LabHome/TravelGuidelines.md [hunk 2]
```diff
@@ -7,16 +7,16 @@ nav_order: 8
...
+As much as possible, all flights + hotels should be booked via Concur, which avoids carrying significant costs on your personal credit card. Please submit receipts _immediately_ after the trip. Delayed submission can risk the ability to receive reimbursement due to strict university/federal regulations, which is a bureaucratic process.  Any expenses not approved for reimbursement (even if submitted before!) within 6 months are NOT eligible for reimbursement. Reimbursement is via Concur; please see the [Concur](https://pennlinc.github.io/docs/LabHome/CONCUR_Reimbursement/) documentation and check the #Concur channel for details.
```
Comment (kahinimehta) [inline docs/LabHome/TravelGuidelines.md:20]: probably worth updating the concur docs

File: docs/documentation/documentation_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.bash_profile
+```
+
+Now install `jekyll` using a ruby "gem", by doing the following:
+
+```
+# PERMISSIONS MAY BE REQUIRED
+
+gem install --user-install bundler jekyll
```
Comment (kahinimehta) [inline docs/documentation/documentation_guidelines.md:187]: This didn't work for me - lots of version/ dependency issues

File: docs/LabHome/TravelGuidelines.md (hunk 2 above)
Comment (tsalo) [reply docs/LabHome/TravelGuidelines.md:20]: I don't think I actually updated this section. I think VSCode just automatically removed trailing spaces.

File: docs/documentation/documentation_guidelines.md (hunk 3 above)
Comment (tsalo) [reply docs/documentation/documentation_guidelines.md:187]: Yeah I didn't use these instructions either. The changes here are just from me moving the files.

**Review by kahinimehta — APPROVED** [review]

**Pull request merged**